generate_stats.py

```diff
@@ -32,11 +31,10 @@ def main():
 
     # get tags
     dagmc_tags = dagmc_stats.get_dagmc_tags(my_core)
-    
+    print(dagmc_tags)
```

Comment:
I think this is just from debugging, please delete.

There may be value in adding a method to write a summary of the tags.

Reply:
But writing out info about tags should probably be a different PR